### PR TITLE
cli.main: Change default log format, add color if available

### PR DESCRIPTION
**Pull request by nbraud**
Follow-up on #54

### CODE REVIEW

**Comment by duckinator** [comment]
@nbraud is this meant to be _instead of_ #54 or _in addition to_ it (after merging 54 + rebasing this off master)? It appears to be a superset of it, so not entirely sure which approach you're aiming for.

**Comment by astronouth7303** [comment]
That's probably just github's UI when you branch off a branch and submit both as PRs.

**Comment by nbraud** [comment]
In addition to.

Technically, this shouldn't need rebasing before merge, but I will rebase anyhow because otherwise Github won't refresh the displayed diff...

**Comment by duckinator** [comment]
So, #54 should be merged in a little bit.

File: bork/cli.py [hunk 1]
```diff
@@ -67,9 +70,25 @@ def main():
     if verbose:
         sys.argv.remove('--verbose')
 
+    try:
+        import coloredlogs
```
Comment (duckinator) [inline bork/cli.py]: This should solve the linting problem:

```suggestion
        import coloredlogs  # pylint: disable=import-outside-toplevel
```

**Comment by nbraud** [comment]
Picked-up your lint fix and rebased.

**Pull request merged**